src/systems/ec/implementations/vanilla/packages/ec-component-button/ec-component-button-print.scss

```diff
@@ -0,0 +1,130 @@
+/**
+ * Button component
+ * @define button
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-button-print(
+  $border-width: 1.5pt,
+  $padding-horizontal: $ecl-spacing-print-s,
+  $padding-vertical: $ecl-spacing-print-m
+) {
+  .ecl-button {
+    appearance: none;
+    background: none;
+    border-width: 0;
+    box-sizing: border-box;
+    display: inline-block;
+    font: $ecl-font-print-m;
+    font-weight: $ecl-font-weight-print-bold;
+    margin: 0;
+    padding: $padding-horizontal $padding-vertical;
```

Comment:
I'd also remove the padding on the button in print ;)

Reply:
I'm not sure...
Label is hardly readable without padding